lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -84,7 +84,8 @@ function dosomething_northstar_register_user($form_state) {
 function _dosomething_northstar_build_http_client() {
   $base_url = NORTHSTAR_URL;
   if (getenv('DS_ENVIRONMENT') === 'local') {
-      $base_url .=  ":" . NORTHSTAR_PORT;
+      // $base_url .=  ":" . NORTHSTAR_PORT; this prints http://northstar-qa.dosomething.org:8000
```

Comment:
that's from pulling from stage, you can reset that locally [here](http://dev.dosomething.org:8888/admin/config/services/northstar)